credsweeper/credentials/line_data.py

```diff
@@ -250,7 +250,7 @@ def is_source_file_with_quotes(self) -> bool:
         """
         if not self.path:
             return False
-        if Util.get_extension(self.path) in self.config.source_quote_ext:
+        if Util.get_extension(self.path).lower() in self.config.source_quote_ext:
             return True
         return False
```

Comment:
How about add default option parameter `lower=True` to `Util.get_extension()`?
I think it would be good.